src/junox/junox.js

```diff
@@ -34,14 +36,16 @@ export default class Junox {
   }
 
   noteOn (note, velocity) {
-    this.lastNoteOn = note
     const voiceIndex = this.voices.findIndex(voice => voice.note === note)
     const newVoice = new Voice({
       note,
       patch: this.patch,
       velocity,
       sampleRate: this.sampleRate
     })
+    if (!this.voices.length) {
```

Comment:
I check the Juno 60 manual, it says indeed that when there are no voices, the lfo triggers